### ISSUE TITLE
Paths in the verbose mode.

### ISSUE THREAD

**Issue by mity** [mity/acutest#32]
TL;DR: With the `--verbose` option, I am not happy with the too long paths to the source files in Acutest output.

Normally, Acutest outputs only `basename(__FILE__)`. In verbose mode it currently outputs complete `__FILE__`.

However it does not have a consistent behavior because it is highly build system dependent. E.g. when using `cmake -G "Unix Makefiles"` it leads to absolute paths, with `cmake -G "Ninja"` there are relative paths from the build dir. I guess various unnamed IDEs also may have some funny ideas how exactly they launch the compiler.

In either case it often makes the `--verbose` output much harder to parse for human eyes.

<details>
<summary>For illustration...</summary>

```
Test empty:
  /home/mity/prj/c-reusables/tests/test-rbtree.c:110: Check rbtree_verify(&tree) == 0... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:111: Check rbtree_is_empty(&tree)... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:112: Check rbtree_insert(&tree, make_val(42), val_cmp) == 0... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:113: Check rbtree_verify(&tree) == 0... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:114: Check !rbtree_is_empty(&tree)... ok
  /home/mity/prj/c-reusables/tests/test-rbtree.c:116: Check rbtree_is_empty(&tree)... ok
  SUCCESS: All conditions have passed.

Test insert:
  Case Ascending order:
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:154: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:156: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:158: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
  Case Descending order:
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:154: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:156: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:158: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
  Case Randomized order:
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:70: Check v != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:142: Check rbtree_insert(&tree, make_val(values[i]), val_cmp) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:143: Check rbtree_verify(&tree) == 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:149: Check rbtree_lookup(&tree, &key.the_node, val_cmp) != NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:154: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:156: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:158: Check rbtree_lookup(&tree, &key.the_node, val_cmp) == NULL... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
    /home/mity/prj/c-reusables/tests/test-rbtree.c:163: Check rbtree_insert(&tree, &tmp.the_node, val_cmp) != 0... ok
  SUCCESS: All conditions have passed.

Summary:
  Count of all unit tests:        2
  Count of run unit tests:        2
  Count of failed unit tests:     0
  Count of skipped unit tests:    0
SUCCESS: All unit tests have passed.
```

</details>

Therefore, I am currently thinking about making to just output `basename(__FILE__)` even in the verbose output. It may be small change in the code but possibly with bigger impact on usability. So the question of the day is: 

Does anyone find the current behavior useful?
Or perhaps someone has a viable counter-proposal?

**Comment by frithrah** [comment]
Thanks for a brilliant library.

One snag with this change: Vim can't find the file when showing test results in the Quickfix window. Disabling the path removal block solves the problem but I agree with you that in any other situation it's good to shorten the file names. In the Vim case however it actively breaks the results. Maybe a flag is the best option.
